qemu/tests/snapshot_negative_test.py

```diff
@@ -0,0 +1,53 @@
+import re
+
+from avocado.utils import process
+
+from virttest import data_dir
+from virttest.qemu_storage import QemuImg
+
+
+def run(test, params, env):
+    """
+    Negative test.
+    Verify that qemu-img supports to check the options used for
+    creating external snapshot, and raise accurate error when
+    specifying a wrong option.
+    1. It should be failed to create the snapshot when specifying
+       a wrong format for backing file.
+    2. It should be failed to create the snapshot when specifying
+       a non-existing backing file.
+    3. It should be failed to create the snapshot when specifying
+       an empty string for backing file.
+
+    :param test: Qemu test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    def _check_command(cmds):
+        """run the command and check the output"""
+        cmds = cmds.split(";")
+        for qemu_img_cmd in cmds:
+            if qemu_img_cmd_agrs:
+                qemu_img_cmd %= qemu_img_cmd_agrs
+            cmd_result = process.run(qemu_img_cmd, ignore_status=True,
+                                     shell=True)
+        if not re.search(err_info, cmd_result.stderr.decode(), re.I | re.M):
```

Comment:
need the indentation here? align the 'if' with cmd_result?

Reply:
Yes, you are right. Updated, many thanks.